benchmarks/bvh_driver/bvh_driver.cpp

```diff
@@ -200,24 +199,6 @@ void BM_radius_search(benchmark::State &state)
     auto const end = std::chrono::high_resolution_clock::now();
     std::chrono::duration<double> elapsed_seconds = end - start;
     state.SetIterationTime(elapsed_seconds.count());
-
-    if (first_pass)
-    {
-      auto offset_clone = ArborX::clone(offset);
-      ArborX::adjacentDifference(offset, offset_clone);
-      double const max = ArborX::max(offset_clone);
-      double const avg = ArborX::lastElement(offset) / n_queries;
-      auto offset_clone_subview = Kokkos::subview(
-          offset_clone, std::make_pair(1, offset_clone.extent_int(0)));
-      double const min = ArborX::min(offset_clone_subview);
-
-      std::ostream &os = std::cout;
-      os << "min number of neighbors " << min << "\n";
-      os << "max number of neighbors " << max << "\n";
-      os << "avg number of neighbors " << avg << "\n";
-
-      first_pass = false;
-    }
```

Comment:
The print to the screen is indeed annoying but yanking it out means we are likely to have to redo it by hand in the future if one wants to use buffer optimization.

Reply:
a) it is in the history if we need it; b) retaining it would likely require separating benchmark and bvh_driver output streams, and complicating the state to pass the ostream. I chose the easy way.